Examples/Website/templates/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
Same as above, but putting here as well that we can't have the website included as-is.